pkg/cloudprovider/fake/cloudprovider.go

```diff
@@ -109,6 +109,8 @@ func (c *CloudProvider) GetInstanceTypes(_ context.Context, _ *v1alpha5.Provider
 		NewInstanceType(InstanceTypeOptions{
 			Name:         "arm-instance-type",
 			Architecture: "arm64",
+			CPU:          resource.MustParse("16"),
```

Comment:
I am curious why do we need to specify the CPU and memory here?

Reply:
I can only verify what it gets scheduled to, so I bumped the cpu/memory up and set a corresponding request in the pod to force it to either fit this instance or not be scheduled.  Now with the filtering, it ends up not scheduled.